packages/salesforcedx-vscode-aura/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "salesforcedx-vscode-aura",
+  "displayName": "Lightning",
+  "description": "Provides code-editing features for Lightning",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "icon": "images/VSCodeLWC.png",
+  "galleryBanner": {
+    "color": "#ECECEC",
+    "theme": "light"
+  },
+  "version": "45.6.0",
+  "publisher": "salesforce",
+  "preview": true,
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.26.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "45.6.0",
+    "chalk": "^2.4.2",
+    "change-case": "^3.1.0",
+    "lightning-lsp-common": "1.0.4",
+    "lwc-language-server": "1.8.3",
+    "opn": "^5.4.0",
+    "vscode": "1.1.17",
```

Comment:
If this dependency is being used exclusively for testing then we can move it to `devDependencies`.